ESSArch_Core/maintenance/tests/test_views.py

```diff
@@ -43,6 +46,133 @@ def test_authenticated_with_permission(self):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
 
 
+class ConversionJobViewSetPreviewTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.user = User.objects.create(username='user')
+        self.conversion_job = ConversionJob.objects.create()
+        self.url = reverse('conversionjob-preview', args=(self.conversion_job.pk,))
+
+    def test_unauthenticated(self):
+        response = self.client.get(self.url, {'name': 'foo'})
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    @mock.patch('ESSArch_Core.maintenance.views.get_conversion_job_preview_response')
+    def test_authenticated(self, mock_get_conversion_job_preview_response):
+        mock_get_conversion_job_preview_response.return_value = Response(status=status.HTTP_200_OK)
```

Comment:
What is this actually testing? Mocking it to return a 200 and then testing that the mock returns 200 seems unnecessary 🙂